lib/truffle/truffle/patching.rb

```diff
@@ -13,11 +17,16 @@ module Kernel
   alias_method :require_without_truffle_patching, :require
 
   def require(path)
```

Comment:
My first PR of this https://github.com/graalvm/truffleruby/pull/68 (rejected) modified the require method internally which I don't think is as nice as the current `pathching.rb`.  I think @eregon had another idea how to do this: https://github.com/graalvm/truffleruby/pull/68#issuecomment-280852060

I'm hoping that these patches would be a short term fix so we'd be able to remove them eventually anyways.